doc/content/components/_index.md

```diff
@@ -2,4 +2,23 @@
 title: "Components"
 description: ""
 weight: 4
+menu:
+  main:
+    weight: 4
 ---
+
+TTN Stack is composed of components. The core components Network Server, Application Server and Join Server follow the LoRaWAN Network Reference Model. TTN Stack also contains an Identity Server, Gateway Server and Console.
+
+{{< figure src="components.png" alt="TTN Stack components" width="80%" height="80%" >}}
```

Comment:
Not sure how I feel with having AWS, Azure and GCP and so prominent in the diagram when we don't even have them.

Reply:
We're very close to this with #1011 and some docs, so we support at minimum AWS Lambda, Azure Functions and Google Cloud Functions